tests/php/Core/ObjectTest.php

```diff
@@ -538,4 +540,26 @@ public function testInjectedExtensions()
         $this->assertInstanceOf(TestExtension::class, $object->getExtensionInstance($mockClass));
         $this->assertInstanceOf($mockClass, $object->getExtensionInstance($mockClass));
     }
+
+    /**
+     * @expectedException InvalidArgumentException
+     */
+    public function testExtensionsAppliedViaANonExantServiceAsAControl()
+    {
+        Config::modify()->set(BaseObject::class, 'extensions', ['NotAService']);
+        $failure = BaseObject::create();
```

Comment:
This doesn't need to be an assignment - `$failure` is irrelevant

Reply:
That is true. But It is also of no consequence. It does serve as an indication. I could swap it out for a comment (or nothing) if you prefer.

Reply:
```diff
- $failure = BaseObject::create();
+ BaseObject::create();
```

👍 

IDEs will warn this as irrelevant code otherwise and eventually someone else will remove it 😄

Reply:
ok, fair enough :)